build/ci.go

```diff
@@ -232,6 +232,9 @@ func doInstall(cmdline []string) {
 	// Show packages during build.
 	gobuild.Args = append(gobuild.Args, "-v")
 
+	// Add -ldl flags for libscroll_zstd.a.
+	gobuild.Args = append(gobuild.Args, "-ldflags", "-extldflags -ldl")
```

Comment:
added this line because I have not found a good way to pass env variables when compiling.